gallery_dl/extractor/lynxchan.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+
+"""Extractors for LynxChan Imageboards"""
+
+from .common import BaseExtractor, Message
+from .. import text
+import itertools
+
+
+class LynxchanExtractor(BaseExtractor):
+    """Base class for LynxChan extractors"""
+    basecategory = "lynxchan"
+
+
+BASE_PATTERN = LynxchanExtractor.update({
+    "kohlchan": {
+        "root": "https://kohlchan.net",
+        "pattern": r"kohlchan\.net"
+    },
+    "endchan": {
+        "root": "https://endchan.org",
+        "pattern": r"endchan\.org"
```

Comment:
```suggestion
        "root": None,
        "pattern": r"endchan\.(?:org|net|gg)",
```
endchan seems to have multiple TLDs
(and also Tor/Lokinet addresses, but lets ignore those for the time being)